test/kubemark/iks/util.sh

```diff
@@ -159,14 +164,14 @@ function complete-login {
 	echo -n -e "Do you have a federated IBM cloud login? [y/N]${color_cyan}>${color_norm} "
 	read -r ISFED
 	if [[ "${ISFED}" = "y" ]]; then
-		bx login --sso -a "${REGISTRY_LOGIN_URL}"
+		ibmcloud login --sso -a "${REGISTRY_LOGIN_URL}"
 	elif [[ "${ISFED}" = "N" ]]; then
-		bx login -a "${REGISTRY_LOGIN_URL}"
+		ibmcloud login -a "${REGISTRY_LOGIN_URL}"
 	else
 		echo -e "${color_red}Invalid response, please try again:${color_norm}"
 		complete-login
 	fi
-	bx cr login
+	ibmcloud cr login
```

Comment:
For the future - it would be useful to separate changes like s/bx/ibmcloud/ to a separate PR :-)

Reply:
Absolutely. Will do next time.